### PR TITLE
[WB-3070] Add support for beta distributions in Sweeps

### PR DESCRIPTION
**Pull request by charlesfrye**
**tl;dr**: Motivated by a user suggestion, I've added a distribution to Sweeps that is useful for random searches involving DropOut probability and similar parameters.

Details [on JIRA](https://wandb.atlassian.net/browse/WB-3070).

You can see an example of this in action: one project where the parameters are set to [produce a uniform distribution](https://app.wandb.ai/charlesfrye/beta-sweeps/sweeps/wnbsypzh?workspace=user-charlesfrye) and another where they are set to [produce a bimodal distribution with peaks at 0 and 1](https://app.wandb.ai/charlesfrye/beta-sweeps/sweeps/n0n21b2a?workspace=user-charlesfrye).

I also locally checked the other two added code paths: one for confirming that the parameters are valid and another for producing a `q_`uantized version of this distribution.

I couldn't quite figure out the testing/`tox` formatting setup, but I confirmed that my lines didn't add any linter complaints.

### CODE REVIEW

**Comment by vanpelt** [comment]
This is awesome!  @raubitsj do we need to get these changes into anaconda as well?  @charlesfrye if you merge master the tests will pass, we'll want these in the `client-ng` repo and update the docs.

**Comment by cvphelps** [comment]
Blocked on anaconda testing for sweeps

**Comment by dannygoldstein** [comment]
Closing this as it's now implemented in wandb/sweeps#6.
